report/2_background.txt

```diff
@@ -0,0 +1,10 @@
+Serious games are applicable for education in a very wide range of areas including health (such as for the treatment of ADHD (FDA, 2020)), military training and recruitment (America’s Army (U.S Army, 2020)), and education, as we shall discuss in this report. 
+Gamification
+University lecturers are increasingly using some form of gamification during lectures as a form of motivation and consolidation for students. At a university, it was found that 32% of lecturers were using a version of gamification in lectures and of the lecturers that were not, 92% stated they were most likely to use it in the near future (Karmela Aleksic-Maslac, 2017). University students, particularly engineering students, are afraid to feel ashamed by participating in lectures by asking questions or giving the wrong answers to questions. Therefore, the use of gamification in a lecture allows students to engage in their courses without the impact of real-world human relationships. (Asako Ohno, 2013)
+Existing Solutions
+Gamification has been used in education to great effect in the past. One such example is the study done to the Vienna University of Technology (Adrian Dabrowski, 2014). In this paper, the entire course structure was adapted into a gamified version. In addition to ordinary lectures that would still be attended, students were expected to complete a range of “challenges”. These challenges were designed to fit into a “movie style” hacking environment to make the experience of completing the challenges more entertaining, and potentially easier to remember. The study also took advantage of the progress reporting and competition gamification techniques by providing a leaderboard for students on the course. The study also made use of achievements and badges, also a well-known gamification technique, to motivate students to do well. The study found that 96% of the students that took the course enjoyed the “gaming-like aspects” of the practical exercise. 57% also responded that gamification of topics would help to encourage students to take cyber security modules and courses. 
+Another study at Leeds Beckett University (Z. Cliffe Schreuders, 2016) which used gamification as a method to improve motivation for completing learning tasks beyond assignments. The methodology involved the creation of three kinds of experience points (XP), with each focusing on a different element of the course: practical skills, knowledge, and reflection. Tasks to be completed were given XP values and were framed as “quests”. Students were also given access to an online tool that allowed them to track the amount of each XP they had accrued at the year progressed. All students responded that they liked having access to their progress at any time, with 63% liking seeing their relative XP compared to other students. All students taking part in the course also said that they did more out of class work than they would have done otherwise. 
```

Comment:
```suggestion
Another study at Leeds Beckett University (Z. Cliffe Schreuders, 2016) which used gamification as a method to improve motivation for completing learning tasks beyond assignments. The methodology involved the creation of three kinds of experience points (XP), with each focusing on a different element of the course: practical skills, knowledge, and reflection. Tasks to be completed were given XP values and were framed as “quests”. Students were also given access to an online tool that allowed them to track the amount of each XP they had accrued at the year progressed. All students responded that they liked having access to their progress at any time, with 63% liking seeing their relative XP compared to other students. All students taking part in the course also said that they did more out-of-class work than they would have done otherwise. 
```